src/id_token_provider.erl

```diff
@@ -96,10 +99,17 @@ refresh(Provider) ->
 
 refresh(Provider, WellKnownUri) ->
   {ok, KeysUrl} = id_token_jwks:get_jwks_uri(WellKnownUri),
```

Comment:
This one will also fail and crash if the provider is not available, so the error would need to be handled as well

Reply:
good point! fixed